docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -672,6 +688,261 @@ This element contains the settings for the widget class. After adding this eleme
    <support-size>.....</support-size>
 </widget-class>
 ```
+### \<component-based-application\> Element
+
+This element contains the manifest information for a Tizen native Component-based application.
+
+For more information on the relationship between the elements, see the [element hierarchy](#hierarchy).
+
+**Occurrences:**
+
+- 1 or more (optional)
+
+**Expected children (in the following order):**
+
+| Child element           | Occurrences          |
+|-------------------------|----------------------|
+| `<label>`               | 1 or more (optional) |
+| `<icon>`                | 1 or more (optional) |
+| `<app-control>`         | 1 or more (optional) |
+| `<metadata>`            | 1 or more (optional) |
+| `<datacontrol>`         | 1 or more (optional) |
+| `<background-category>` | 1 or more (optional) |
+| `<splash-screens>`      | 1 or more (optional) |
+| `<frame-component>`     | 1 or more (mandatory)|
+| `<service-component>`   | 1 or more (mandatory)|
+
+**Attributes:**
+
+- `appid`
+
+  Application unique ID (string)
```

Comment:
Done

Reply:
Please fix the output:

![appid](https://user-images.githubusercontent.com/35525588/67077027-f72af500-f1ab-11e9-80dd-cd961ddae93b.PNG)

Reply:
@hjhun you need to add \\ in front of \~ to prevent the effect.

```
You can use the a\~z, A\~Z, 0\~9, ".", "-", and "\_" characters, and the value must be less than 50 characters.
```

Reply:
Done